### ISSUE TITLE
Not enough colors and color sequence

### ISSUE THREAD

**Issue by Siggenx** [copperwire/SIMS#7]
We need to add more colors. I replaced line 80 with an example:

``` Python
colour_list = ['blue', 'green', 'red', 'purple', 'cyan', 'black', 'orange',  'gray', 'firebrick', 'navy']
```

In addition, I believe it is better to have the same sequence for all plots instead of randomizing the colors. This is also what they do in the matplotlib library, and if there is not enough colors defined, the sequence begins at the start again.

**Comment by copperwire** [comment]
This is a recognized problem and is fixed in version 1.0 which is set to be uploaded early next week.

**Comment by copperwire** [comment]
Well I changed the version name, but this feature is also corrected.
